epub34/authoring/index.html

```diff
@@ -5829,6 +5828,410 @@ <h4>The <code>itemref</code> element</h4>
+				<h3>Layout types</h3>
+
+				<section id="sec-reflowable">
+					<h4>Reflowable layouts</h4>
+
+					<p>A reflowable layout is one where the contents of [=EPUB content documents=] are reflowed to fit
+						the available space in the [=reading system=] viewport. A reading system can then dynamically
+						paginate the content or provide it in a scrolled manner.</p>
+
+					<p>Reflowable layouts are the default for EPUB 3 and were historically the only way its predecessor
+						formats were laid out. Because they are the default, it is not required to declare any metadata
+						to state that an [=EPUB publication=] is intended to be reflowed. If an explicit declaration is
+						preferred, however, the <a href="#layout"><code>rendition:layout</code> property</a> MAY be set
+						in a <code>meta</code> element that specifies the value <dfn id="def-layout-reflowable"
+								><code>reflowable</code></dfn>.</p>
+
+					<aside class="example" title="Declaring an EPUB publication is reflowable">
+						<pre>&lt;package &#8230;>
+   &lt;metadata>
+      &lt;meta property="rendition:layout">reflowable&lt;/meta>
+      &#8230;
+   &lt;/metadata>
+   &#8230;
+&lt;/package></pre>
+					</aside>
+
+					<p>In addition to setting an entire EPUB publication as reflowable, it is possible to override a <a
+							href="#sec-pre-paginated">pre-paginated layout</a> to specify that some spine items are
+						reflowable. To do so, the <a href="#layout-reflowable"><code>rendition:layout-reflowable</code>
+							spine override</a> MUST be set in the <code>properties</code> attribute on each spine
+							<code>itemref</code> that is reflowable.</p>
+
+					<aside class="example" title="Reflowable document in a pre-paginated layout">
+						<pre>&lt;package &#8230;>
+   &lt;metadata>
+      &lt;meta property="rendition:layout">pre-paginated&lt;/meta>
+      &#8230;
+   &lt;/metadata>
+   &lt;manifest>
+      &lt;item id="reflow1" href="preface.html" media-type="application/xhtml+xml"/>
+   &lt;/manifest>
+   &lt;spine>
+   	&lt;itemref idref="reflow1" properties="rendition:layout-reflowable"/>
+   &lt;/spine>
+&lt;/package></pre>
+					</aside>
+
+					<div class="note">
+						<p><a href="#sec-roll">Roll layouts</a> do not support <a href="#layout-overrides">layout spine
+								overrides</a> so reflowable layouts cannot be similarly mixed with rolls.</p>
+					</div>
+
+					<p>The following properties from the <a href="#app-rendering-vocab">Package rendering vocabulary</a>
+						can be applied to reflowable spine items to further control their rendering:</p>
+
+					<ul>
+						<li>the <a href="#align-x-center"><code>rendition:align-x-center</code> spine override</a> is
+							used to indicate a preference for having the specified spine item's content be
+							center-aligned by reading systems.</li>
+					</ul>
+
+					<p>All other properties in the Package rendering vocabulary are ignored for reflowable spine
+						items.</p>
```

Comment:
```suggestion
					<p>All other properties in the Package rendering vocabulary are ignored for reflowable spine
						items.</p>
```

Shouldn't we use a more normative language for this type of statement (it is repeated in the other layout subsection). Something like "Other properties in the Package rendering vocabulary _MUST NOT_ be used in spine items". If we are afraid of making some valid documents suddenly invalid, we could used _SHOULD NOT_.

Reply:
Those lists are a placeholder for now until we figure out what properties are even surviving this revision. I had all the other properties listed but then took them out to match what looks like will be left after we cull the vocabulary.

But going back to when the list had global properties in it, not just the remaining spine overrides, the problem is that you can mix layouts using spine overrides. So say I want to globally set a value for all the pre-paginated overrides I've added to my globally reflowable layout. A restriction that you must not or should not set pre-paginated properties in reflowable layouts would make it invalid and you'd have to repeat spine overrides everywhere. They're fine to set but they're ignored for reflowable documents by reading systems... hence the statement.

There's also the problem that warnings are as bad as errors for vendor ingestion, so it's probably better to leave the requirement to ignore other properties on reading systems than restrict their presence.